docs/docs/kds-rest-api.md

```diff
@@ -152,6 +150,8 @@ keys respectively). The API version field will contain the API version used for
             (BE)    sized field             (BE)   sized field
 ```
 
+The API version field contains the API version used for the request. The master key type field contains the type ID 
+of the master key used for derivation (0, 1, and 2 for development, cluster, and Azure key vault, respectively). 
```

Comment:
The values for the type should be in a numbered list.

Reply:
Added a table instead in [another PR](https://github.com/R3Conclave/conclave-core-sdk/pull/50).